recipes/radio-beam/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "radio_beam" %}
+{% set version = "0.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3b3b3cfcc17d61df469c9e45d7f7fc993b902f3f2c076279071fd984a6e5763d" %}
+
+package:
+  name: radio-beam
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  '{{ hash_type }}': '{{ hash_value }}'
+  patches:
+    - offline.patch
+
+build:
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  noarch: python
+
+requirements:
+
+  build:
+    - python
+    - pip
+    - setuptools
+
+  run:
+    - python
+    - astropy
+    - numpy >=1.9.0
+
+test:
+  imports:
+    - radio_beam
+
+about:
+  home: https://github.com/radio-astro-tools/radio-beam
+  license: BSD-3-Clause
```

Comment:
This can be fixed in the feedstock since you need to get this merged, but I think we usually use just `3-Clause` here since `BSD` is in the license family.

Reply:
`BSD-3-Clause` (with `BSD` explicit) is ok. We have typically included it.

Reply:
@jakirkham It would be nice to have a documented standard here, maybe [this](https://opensource.org/licenses/alphabetical) is a good source.

Reply:
oh yes, silly me, that's exactly the documented reference in the example `meta.yaml`...